src/module/cache-container/view/cache-container.html

```diff
@@ -75,7 +75,7 @@ <h1><span class="label"
     <div class="col-sm-12 col-md-12">
       <ul class="nav nav-tabs">
         <li ui-sref-active="active" ui-sref=".caches"><a translate>CACHES</a></li>
-        <li ui-sref-active="active" ui-sref=".counters" ng-if="!ctrl.isLocalMode()"><a translate>COUNTERS</a></li>
+        <li ui-sref-active="active" ui-sref=".counters"><a translate>COUNTERS</a></li>
```

Comment:
This is strange, even though this looks correct when I load up the server with `standalone.sh` no counters tab is present.

Reply:
Try using incognito windows...I'll try again myself.